package.json

```diff
@@ -29,7 +29,7 @@
   },
   "repository": {
     "type": "git",
-    "url": "https://github.com/strongloop/loopback-connector-mongodb"
+    "url": "https://github.com/clarkorz/loopback-connector-mongodb"
   },
   "license": {
     "name": "Dual MIT/StrongLoop",
```

Comment:
Please revert the changes to package.json. The version will be updated by our release tools.